com.uploadedlobster.peek.json

```diff
@@ -21,8 +22,10 @@
     "cflags": "-O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2",
     "cxxflags": "-O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2",
     "ldflags": "-fstack-protector-strong -Wl,-z,relro,-z,now",
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
     "env": {
-      "V": "1"
+      "V": "1",
+      "CARGO_HOME": "/run/build/gifski/cargo"
```

Comment:
Is `CARGO_HOME` actually needed? At least for my local builds using `org.freedesktop.Sdk.Extension.rust-stable` I could build without it